docs/snapshot.md

```diff
@@ -34,6 +41,26 @@ parameters:
 }
 ```
 
-## Failure Mode
+## Failure Mode 
 
 The driver will attempt to check if the availability zones provided are supported for fast snapshot restore before attempting to create the snapshot. If the `EnableFastSnapshotRestores` API call fails, the driver will hard-fail the request and delete the snapshot. This is to ensure that the snapshot is not left in an inconsistent state.
+
+
+# Amazon EBS Local Snapshots on Outposts
+
+The EBS CSI Driver provides support for [Amazon EBS local snapshots on Outposts](https://docs.aws.amazon.com/ebs/latest/userguide/snapshots-outposts.html) via `VolumeSnapshotClass.parameters.outpostArn`.
+
+By default, snapshots of EBS volumes on an Outpost are stored in Amazon S3 in the Region of the Outpost. You can also use Amazon EBS local snapshots on Outposts to store snapshots of volumes on an Outpost locally in Amazon S3 on the Outpost itself. This ensures that the snapshot data resides on the Outpost, and on your premises. In addition, you can use AWS Identity and Access Management (IAM) policies and permissions to set up data residency enforcement policies to ensue that snapshot data does not leave the Outpost. This is especially useful if you reside in a country or region that is not yet served by an AWS Region and that has data residency requirements.
```

Comment:
np: Suggest removing this. This is reiterating what's already in the doc.

Reply:
We do the same for fast snapshot. The # Fast Snapshot Restores section in the first paragraph of the linked doc. Would you suggest we remove this from both places and just have a link to the doc ?

Reply:
Agree - if there is information verbatim copied from the EBS docs, it should be removed (and replaced with a link).